### PR TITLE
Update readme and add PR template

### PR DESCRIPTION
**Pull request by saturninoabril**
#### Summary
Update readme:
- Added ``How to Test Notices via `/cloud` Plugin`` to serve as guide primarily for those submitting a PR to ensure that the notice works as expected, and  for anyone reviewing a PR.
- Added `QA criteria when reviewing a PR`.
- Fixed other typo errors.

Add PR template:
- Set the base requirement when submitting a PR.

#### Ticket Link
none

P.S. I've added many reviewers to get buy in and approval from everyone on what is expected when submitting and reviewing a PR in this repo. Goal is for quality and testing to start with the PR submitter and ensure that the notice functionality is as expected. Notice testing can be very much involved. However, the burden should not entirely leave into the hands of QA alone. It should be shared.

PR submitter is expected to:
- ensure that the notice functionality works as expected.
- provide screenshots for ease of review.
- request for review for those implementing the feature to be announced.

Reviewers:
- PM and Technical writer for content.
- QA review for validating aesthetic and functionality.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  A description of this pull request.
+-->
+
+#### Jira ticket
+- MM-Txxxxx
+
+#### Screenshots of the modals or screens in all target clients (required)
+- [] In Web
+- [] In Desktop app
+- [] In Mobile app, both Android and iOS
```
Comment (amyblais) [inline .github/PULL_REQUEST_TEMPLATE.md:13]: Are there guidelines for how or where I can capture screenshots for new notices?

**Review by cwarnermm — APPROVED** [review]
Great updates, @saturninoabril!

**Review by esethna — APPROVED** [review]
Thanks Saturn for tracking this down and documenting it, super appreciated. It looks good

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (esethna) [reply .github/PULL_REQUEST_TEMPLATE.md:13]: Correct, it's not implemented on mobile yet

**Review by stevemudie — APPROVED** [review]
This looks great @saturninoabril Thank you for this!

File: README.md [hunk 2]
```diff
@@ -12,43 +12,43 @@ Notices are modals that appear in-product when users meet certain critera as def
 
 Notices can be triggered on the following conditions:
 
-**External Dependency**: If an external dependency going to be deprecated for a certain version.
+- **External Dependency**: If an external dependency going to be deprecated for a certain version.
```
Comment (lindalumitchell) [inline README.md:15]:
```suggestion
- **External Dependency**: If an external dependency is going to be deprecated for a certain version.
```

**Review by lindalumitchell — APPROVED** [review]
This is terrific, Saturn, thank you for doing this! Looks great to me (just one very minor suggestion).

**Review by amyblais — APPROVED** [review]

**Pull request merged**